### PR TITLE
Updated dynamic demo plugin instructions

### PR DESCRIPTION
**Pull request by sabre1041**
Update and alignment of Dynamic Demo Plugin instructions to align with [console-plugin-template](https://github.com/spadgett/console-plugin-template)

### CODE REVIEW

File: dynamic-demo-plugin/README.md [hunk 1]
```diff
@@ -28,6 +30,29 @@ the script, for example:
...
 ```
 
+In another terminal window, run:
+
+1. `oc login` (requires [oc](https://console.redhat.com/openshift/downloads) and an [OpenShift cluster](https://console.redhat.com/openshift/create))
+2. `yarn run start-console` (requires [Docker](https://www.docker.com) or [podman 3.2.0+](https://podman.io))
+
+This will run the OpenShift console in a container connected to the cluster
+you've logged into. The plugin HTTP server runs on port 9001 with CORS enabled.
+Navigate to <http://localhost:9000/example> to see the running plugin.
```
Comment (spadgett) [inline dynamic-demo-plugin/README.md:40]: Does the demo plugin have a different URL or does it also use this route?

**Review by spadgett — COMMENTED** [review]
Thanks @sabre1041 !

File: dynamic-demo-plugin/README.md (hunk 1 above)
Comment (sabre1041) [reply dynamic-demo-plugin/README.md]: @spadgett It was a copy/paste error from the template repository. Has been updated

**Review by spadgett — APPROVED** [review]
/lgtm

**Comment by spadgett** [comment]
/label docs-approved
/label px-approved
/label qe-approved

This PR only changes the README and adds a dev script. No changes to console itself.

**Pull request merged**